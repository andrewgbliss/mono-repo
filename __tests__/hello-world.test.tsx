import { render } from '@testing-library/react';
import HelloWorld from '../pages/hello-world';

it('renders hello world unchanged', () => {
  const { container } = render(<HelloWorld />);
  expect(container).toMatchSnapshot();
});
