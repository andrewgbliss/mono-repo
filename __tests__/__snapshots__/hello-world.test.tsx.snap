// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders hello world unchanged 1`] = `
<div>
  <div>
    Hello World
  </div>
</div>
`;
